### PR TITLE
Fixes runtime error with custom notifiers.

### PR DESCRIPTION
**Pull request by fusiondog**
This addresses the bug in https://github.com/AcalephStorage/consul-alerts/issues/96

### CODE REVIEW

File: consul/client.go [hunk 1]
```diff
@@ -334,7 +334,13 @@ func (c *ConsulAlertClient) NewAlerts() []Check {
...
 	if kvPairs, _, err := c.api.KV().List("consul-alerts/config/notifiers/custom", nil); err == nil {
 		for _, kvPair := range kvPairs {
-			custNotifName := filepath.Base(kvPair.Key)
+			rp := regexp.MustCompile("/([^/]*)$")
+			match := rp.FindStringSubmatch(kvPair.Key)
+			custNotifName := match[1]
+			if custNotifName == "" {
+				continue
+			}
+			customNotifs = make(map[string]string)
```
Comment (ariscn) [inline consul/client.go]: Shouldn't custNotifName be defined outside the for loop?

Comment (fusiondog) [inline consul/client.go]: Each key from the custom notifiers directory will be a different notifier name.  So the loop will need to set the name of the custom notifier for each entry as it goes through.  Imagine 3 custom notifiers defined in the custom directory with the key name as the name of the notifier and the value is the command to run:
consul-alerts/config/notifiers/custom/
  - "SMS" -> SMScommand.sh
  - "script_action" -> somescript.pl
 - "alt_email" -> custom_email_script.rb

The loop needs to set the custNotifName for each to use as a key in the customNotifs map.

Comment (Rhuagh) [inline consul/client.go]: customNotifs can be defined outside the loop though.

Comment (ariscn) [inline consul/client.go]: Yep, wrote the wrong variable name earlier, I meant customNotifs.

Comment (fusiondog) [inline consul/client.go]: Good call new commit pushed.

**Pull request merged**